nni/algorithms/compression/v2/pytorch/pruning/iterative_pruner.py

```diff
@@ -227,6 +224,11 @@ class SimulatedAnnealingPruner(IterativePruner):
         Cool down rate of the temperature.
     perturbation_magnitude : float
         Initial perturbation magnitude to the sparsities. The magnitude decreases with current temperature.
+    pruning_algorithm : str
+        Supported pruning algorithm ['level', 'l1', 'l2', 'fpgm', 'slim', 'apoz', 'mean_activation', 'taylorfo', 'admm'].
+        This iterative pruner will use the chosen corresponding pruner to prune the model in each iteration.
+    pruning_params : Dict
+        If the pruner corresponding to the chosen pruning_algorithm has extra parameters, put them as a dict to pass in.
```

Comment:
"the pruner corresponding to the chosen pruning_algorithm" -> "the chosen one-shot pruner"

Reply:
same as previous

Reply:
-> If the chosen pruning_algorithm has extra parameters,

Reply:
fix